app/konnect/mesh-manager/service-mesh.md

```diff
@@ -3,7 +3,9 @@ title: Configure a Mesh Global Control Plane with the Kubernetes demo app
 content_type: tutorial
 ---
 
-Using Mesh Manager, you can create {{site.mesh_product_name}} global control planes to manage your {{site.konnect_saas}} mesh services. This guide explains how to configure a global control plane with {{site.mesh_product_name}}, then install the Kubernetes demo app so you can start interfacing with {{site.mesh_product_name}} in {{site.konnect_saas}}.
+Using Mesh Manager, you can create global control planes to manage your {{site.mesh_product_name}} meshes. This guide explains how to configure a global control plane, install the Kubernetes demo app so you can start interfacing with {{site.mesh_product_name}} in {{site.konnect_saas}}, and how to apply {{site.mesh_product_name}} policies. WHY KUMACTL.
+
+In this scenario, you work for a clothing company that has an website where customers can shop online. Your clothing company is anticipating a high number of sales for your next clothing launch, so you want to get ahead of this by rate limiting your services so your website doesn't crash due to the increase in traffic. You decide that the best way to manage your services is with a service mesh. <!-- why not a plugin? --> Your company already uses {{site.konnect_short_name}} for other services, so you decided to use {{site.konnect_short_name}} to host your global control plane along with {{site.mesh_product_name}} for your service mesh. 
```

Comment:
Few questions:
- Is a rate limiting policy the best option to use here? Maybe I should use the mTLS scenario that's already in the demo app instructions? https://docs.konghq.com/mesh/latest/quickstart/kubernetes/#enable-mutual-tls-and-traffic-permissions
- Why should a customer use a service mesh policy instead of a Konnect plugin? Like with rate limiting, I'm assuming they do the same thing, so why would you use one over the other?